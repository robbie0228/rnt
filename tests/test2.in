board
move a up  
move a left
move a right
move a down
move A down
move A left
move A right
move A up
board
move a down
move A up
move a down
move A up
board
move a down
board
quit
