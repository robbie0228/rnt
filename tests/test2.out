Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Invalid move

Invalid move

Cannot move a link onto another of your links

Cannot move opponent's piece

Invalid move

Invalid move

Cannot move a link onto another of your links

Cannot move opponent's piece

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
.bcSSfgh
a..de...
........
........
........
........
A..DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
.bcSSfgh
...de...
........
a.......
A.......
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 1D, 0V
Abilities: 5
a: D1   b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
...de...
........
........
a.......
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: D1   B: D2   C: D3   D: D4
E: V1   F: V2   G: V3   H: V4

Player 1:
Downloaded: 1D, 0V
Abilities: 5
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
.bcSSfgh
...de...
........
........
a.......
........
.B.DE...
..CSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: D1   B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

